Documentation/install/upgrade.rst

```diff
@@ -316,6 +316,24 @@ Annotations:
 1.7 Upgrade Notes
 -----------------
 
+1.7.13 Upgrade Notes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A new flag ``--egress-multi-home-ip-rule-compat`` was introduced to allow
+specifying whether to maintain compatibility with the older ENI mode datapath.
+It defaults to ``false`` meaning it will instruct Cilium to **not** maintain
+compatibility and perform a migration to the new ENI datapath, upon Cilium
+upgrade. This operation will modify the routing table to fix a potential
+collision.
```

Comment:
As discussed, would be good to have a step here to verify success of the migration.